corehq/apps/app_manager/static/app_manager/js/releases/app_view_release_manager.js

```diff
@@ -11,11 +11,12 @@ hqDefine("app_manager/js/releases/app_view_release_manager", function() {
         recipient_contacts: initial_page_data('sms_contacts'),
         download_modal_id: '#download-zip-modal',
         fetchLimit: initial_page_data('fetch_limit'),
+        latestReleasedVersion: initial_page_data('latestReleasedVersion'),
     };
     var el = $('#releases-table');
     if (el.length) {
         var releasesMain = releasesMainModel(o);
-        _.defer(function(){ releasesMain.getMoreSavedApps(false); });
+        _.defer(function(){ releasesMain.goToPage(1); });
```

Comment:
Could this go inside `pagination.js`?

Reply:
I don't think that `pagination.js` should make inferences on how data gets loaded into the view on first load (i.e. this page could conceivably load the first page of releases without doing an ajax call).